FWCore/Framework/test/stubs/TestOutputModule.cc

```diff
@@ -83,6 +88,12 @@ namespace edmtest {
 
   TestOutputModule::~TestOutputModule() {}
 
+  Trig TestOutputModule::getTriggerResults(EDGetTokenT<TriggerResults> const& token, EventForOutput const& e) const {
+    Trig result;
+    e.getByToken<TriggerResults>(token, result);
+    return result;
```

Comment:
This could be just
```suggestion
    return e.getHandle(token);
```
right?

Reply:
Done. Thanks. That is better.